server/server/Models/restaurant.js

```diff
@@ -0,0 +1,41 @@
+const mongoose = require("mongoose");
+
+// schema
+const restaurantSchema = new mongoose.Schema({
+  _id: mongoose.Schema.Types.ObjectId,
+
+  name: {
+    type: String,
+    required: true,
+  },
+
+  location: {
```

Comment:
location will be based on longitude and latitude, so it should be an array of only 2 numbers